Modules/GameOptionsSender/PlayerGameOptionsSender.cs

```diff
@@ -100,21 +100,23 @@ public override IGameOptions BuildGameOptions()
                 opt.SetFloat(FloatOptionNames.ImpostorLightMod, 1.25f);
             }
         }
-        switch (role.GetCustomRoleTypes())
+        if (!role.IsGhostRole() || player.IsAnySubRole(x => x is CustomRoles.EvilSpirit)) 
```

Comment:
Incorrectly written conditions?
I mean, does it have to be like this?
`if (!(role.IsGhostRole() || player.IsAnySubRole(x => x is CustomRoles.EvilSpirit)))`

Reply:
Oh wain

Reply:
Forget about it lol

Reply:
ye `AURoleOptions.ShapeshifterCooldown = Options.DefaultShapeshiftCooldown.GetFloat();` was here so it checks for that